src/main.py

```diff
@@ -42,16 +44,30 @@ def __init__(self) -> None:
             help="Flag indicating whether script should not post files to AWS or not",
             default=False,
         )
+        parser.add_argument(
+            "--hard-reset",
+            type=bool,
+            help="Flag indicating whether the existing sync table data should "
+            "be deleted from the bucket and rebuilt from scratch",
+            default=False,
+        )
         arguments, _ = parser.parse_known_args()
         self.sync_table: SyncTable = arguments.sync_table
         self.dry_run: bool = arguments.dry_run
+        self.hard_reset: bool = arguments.hard_reset
 
 
 if __name__ == "__main__":
     load_dotenv()
     volume_path = Path(os.environ["VOLUME_PATH"])
     args = ScriptArgs()
     aws = AWSClient.new_from_environment()
+    if args.hard_reset and not args.dry_run:
+        # Drop Data from AWS bucket
+        aws.delete_all(args.sync_table)
+        # drop backup data from volume path
+        shutil.rmtree(volume_path / str(args.sync_table))
```

Comment:
This syntax is weird, but I guess it works. So we decided to keep our local copy of the data?